atom/common/api/atom_api_crash_reporter.cc

```diff
@@ -31,21 +31,36 @@ struct Converter<CrashReporter::UploadReportResult> {
 
 namespace {
 
+// TODO(2.0) Deprecate
```

Comment:
Maybe TODO should say "Remove" and not "Deprecate" in the next major?
I mean if we introduce new shiny API the old one should become deprecated immediately, not in the next version.